pkgs/tools/misc/dt-shell-color-scripts/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, runtimeShell
+, bash
+, makeWrapper
+}:
+  stdenv.mkDerivation {
```

```suggestion

stdenv.mkDerivation {
```

Remove this indentation.